### ISSUE TITLE
There is no Moodle Mobile App Support

### ISSUE THREAD

**Issue by egp5** [marinaglancy/moodle-format_flexsections#75]
Hi,

Thanks for this very useful plugin. It works great in my Moodle 4.0 instance but when I open the mobile app it appears in the default course format. So I read the Moodle documentation for creating Mobile App support for plugins for the course format plugin and implemented the app support for the Flex Sections plugin. I am adding the screenshot of the app below:

![Screenshot 2023-10-31 at 12 16 24 PM](https://github.com/marinaglancy/moodle-format_flexsections/assets/37616724/39d700d8-d370-45e8-9978-4b0ef876c428)

Since I don't know how to create a recursive function in mustache template, I have implemented only until 3 subsections max.

**Comment by marinaglancy** [comment]
Hello,
thanks for creating this issue. To be honest I have no experience yet adding mobile support to the plugins. To me it sounds that having three nested levels is better than one, we can flatten the deeper sublevels. If you can create a pull request with your solution, it would be great. Thanks

**Comment by egp5** [comment]
Yes, I shall learn about pull request and do it. Kindly give me time.

**Comment by marcelo2108** [comment]
Hello Guys! @egp5, did you have time to solved? we face the same problem. thanks for your amazing job bro.

**Comment by marcelo2108** [comment]
> Hello, thanks for creating this issue. To be honest I have no experience yet adding mobile support to the plugins. To me it sounds that having three nested levels is better than one, we can flatten the deeper sublevels. If you can create a pull request with your solution, it would be great. Thanks

Hello Marina! I noticed a new version was released. Do you know if they were able to resolve this issue with the mobile app? cheers! @marinaglancy

**Comment by marinaglancy** [comment]
Hello, I don't think I will be able to address this myself but I will be happy to review and accept contributions. 

I never worked with mobile app support in plugins myself, I don't even know if add-on course formats CAN change how course looks in the mobile app.

**Comment by marcelo2108** [comment]
Hello Marina, thank you for your response. Don't worry, we'll find a solution eventually. It's a shame that Moodle doesn't include support for this feature. Sincerely!
